routes/Login/login.js

```diff
@@ -66,13 +69,15 @@ router.route('/').post( function(req,res){
             res.status(503).send("ERROR: " + err);
         })
```

Comment:
We want the .catch block to be the last thing that happens before the .done() (I know this was already here, but we want to try and correct these as we come across them.)